browser-test/src/admin/admin_application_statuses.test.ts

```diff
@@ -359,6 +361,171 @@ test.describe('view program statuses', () => {
 
       expect(await adminPrograms.getNoteContent()).toBe(noteText)
     })
+
+    test.describe('email status updates work correctly with PAI flag on', () => {
+      test.beforeAll(async () => {
+        const {page, adminPrograms, adminQuestions} = ctx
+        await enableFeatureFlag(
+          page,
+          'primary_applicant_info_questions_enabled',
+        )
+        await loginAsAdmin(page)
+        await adminQuestions.addEmailQuestion({
+          questionName: 'Email',
+          universal: true,
+          primaryApplicantInfo: true,
+        })
+        await adminPrograms.editProgram(programWithStatusesName)
+        await adminPrograms.editProgramBlock(
+          programWithStatusesName,
+          'block description',
+          ['Email'],
+        )
+        await adminPrograms.publishAllDrafts()
+        await logout(page)
+      })
+      test.beforeEach(async () => {
+        const {page} = ctx
+        // Logout before each test so we can start from the homescreen as an applicant
+        await logout(page)
+      })
```

Comment:
FYI. In the future when I get fixtures going, we don't to do this thing. Each test will be isolated.